contracts/contracts/TACoApplication.sol

```diff
@@ -690,6 +700,7 @@ contract TACoApplication is
      * @notice Get the value of authorized tokens for active providers as well as providers and their authorized tokens
      * @param _startIndex Start index for looking in providers array
      * @param _maxStakingProviders Max providers for looking, if set 0 then all will be used
+     * @param _cohortDuration Duration during which staking provider should be active. 0 means no end date
```

Comment:
Same comment in `TACoChildApplication.sol`
```suggestion
     * @param _cohortDuration Duration during which staking provider should be active. 0 means forever
```

Reply:
✔️